docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: docs
+title: Getting Started
+permalink: /documentation/getting-started/
+---
+# Getting Started
+
+This guide provides step-by-step instructions on how to create a functional web test with TestCafe and contains the following sections.
+
+* [Installing TestCafe](#installing-testcafe)
+* [Creating a Test](#creating-a-test)
+* [Running the Test](#running-the-test)
+* [Viewing the Test Results](#viewing-the-test-results)
+* [Writing Test Code](#writing-test-code)
+  * [Performing Actions on the Page](#performing-actions-on-the-page)
+  * [Observing the Page State](#observing-the-page-state)
+  * [Assertions](#assertions)
+
+## Installing TestCafe
+
+Make sure that [Node.js](https://nodejs.org/) and [npm](https://www.npmjs.com/) are installed on your computer, then run a single command:
+
+```bash
+npm install -g testcafe
+```
+
+## Creating a Test
+
+To create a test, create a new .js file anywhere on your computer.
+This file must have the special structure: tests must be organized into fixtures. So, first you need to declare a fixture by using the [fixture](../test-api/test-code-structure.md#fixtures) function.
+
+You can optionally specify a start page URL for the fixture by using the [page](../test-api/test-code-structure.md#specifying-the-start-webpage) function.
+In this tutorial, you will create a test for the [http://testcafe.devexpress.com/example](http://testcafe.devexpress.com/example) sample page.
+
+Then create the [test](/test-api/test-code-structure.md#tests) function where you will further place test code.
+
+```js
+fixture `Getting Started`
+    .page('http://testcafe.devexpress.com/example');
+
+test('Test1', async t => {
+    // Test code
+});
+```
```

Comment:
I also like the current variant (with growing step-by-step).

I don't want to use comments to describe this code, because comments will litter the test's clear structure. 

Also, it's not good idea to describe all the test elements below the code - too much info at once.